nncf/quantization/algorithms/min_max/algorithm.py

```diff
@@ -335,7 +337,10 @@ def _get_quantization_target_points(self, model: TModel) -> OrderedDict[TargetPo
 
         if self._quantization_target_points_to_qconfig:
             return self._quantization_target_points_to_qconfig
-        quantizer_setup = self._get_quantizer_setup(nncf_graph)
+        backend = get_backend(model)
+        device = self._parameters.target_device
+        pattern = PatternsManager().get_patterns(backend, device).get_full_pattern_graph()
```

Comment:
What do you think about to add `PatternsManager().get_full_pattern_graph(backend, device)` method to avoid double call `PatternsManager().get_patterns(backend, device).get_full_pattern_graph()`?

Reply:
Good suggestion. Added.